### PR TITLE
contrib/intel/jenkins: fabtests ze support

### PR DESCRIPTION
**Pull request by zachdworkin**
Add shm-ze Testing to Jenkins CI

Signed-off-by: zdworkin <zachary.dworkin@intel.com>

### CODE REVIEW

File: contrib/intel/jenkins/build.py [hunk 1]
```diff
@@ -15,27 +15,29 @@
...
-            config_cmd.append('--enable-{}={}'.format(prov, enable_prov_val))
-        for prov in common.disabled_prov_list:
-             config_cmd.append('--enable-{}=no'.format(prov))
-
-        common.run_command(['./autogen.sh'])
-        common.run_command(shlex.split(" ".join(config_cmd)))
-        common.run_command(['make','clean'])
-        common.run_command(['make', '-j32'])
-        common.run_command(['make','install'])
+    if (os.path.exists(libfab_install_path) != True):
```
Comment (aingerson) [inline contrib/intel/jenkins/build.py:18]: Please separate all white space changes into a separate commit. It makes it hard to tell what changes are needed for ZE support and which ones are not.

File: contrib/intel/jenkins/tests.py [hunk 2]
```diff
@@ -211,6 +211,39 @@ def execute_cmd(self, shmem_testname):
         outputcmd = shlex.split(command)
         common.run_command(outputcmd)
 
+class ZeTest(Test):
```
Comment (aingerson) [inline contrib/intel/jenkins/tests.py:214]: Update name to FabtestsZE to distinguish from other ZE testing to be added later

File: contrib/intel/jenkins/tests.py [hunk 3]
```diff
@@ -211,6 +211,39 @@ def execute_cmd(self, shmem_testname):
...
+    def __init__(self, jobname, buildno, testname, core_prov, fabric,
+                 hosts, ofi_build_mode, util_prov=None):
+
+        super().__init__(jobname, buildno, testname, core_prov, fabric,
+                         hosts, ofi_build_mode, util_prov)
+        self.fabtestpath = "{}/bin".format(self.libfab_installpath)
+        self.fabtestzepath = "{}".format(ci_site_config.ze_src)
```
Comment (aingerson) [inline contrib/intel/jenkins/tests.py:221]: I would prefer if we could put the script in the common script directory (currently called mpi_testpath) and rename the variable to script_path or something similar.

**Review by aingerson — COMMENTED** [review]
Also please update commit message to match commit format (contrib/intel: <description>)

**Comment by shefty** [comment]
@aingerson - please merge when you're happy with the updates
